src/components/Catalog.spec.js

```diff
@@ -6,7 +6,19 @@ import '@testing-library/jest-dom';
 jest.mock('./Item', () => () => <div data-testid="Item" />);
 
 let catalog = {};
+let bookingInformation = {};
+let bookingResponse = {};
+let showModal;
+let loadingJourney;
+let handleClose;
+let onSelectJourney;
 describe('Basic functionality', () => {
```

Comment:
Name the testcase as Catalog

Reply:
Updated for this file and other files as well